src/SM/StateMachine/StateMachine.php

```diff
@@ -232,4 +227,36 @@ protected function callCallbacks(TransitionEvent $event, $position)
         }
         return $result;
     }
+
+    /**
+     * @param $transitionName
+     *
+     * @return bool|mixed
```

Comment:
`null|array` maybe?
It's more consistent to return `null` than `false` when there is no matching transition.

Reply:
👍 agreed; it's more semantic this way. Thanks again. I'll include this in an update later today/tonight.